packages/lesswrong/lib/modules/fragments.js

```diff
@@ -62,6 +62,7 @@ registerFragment(`
 
 extendFragment('UsersCurrent', `
   ...UsersMinimumInfo
+  defaultToCKEditor
```

Comment:
shouldn't this just be a "defaultEditor" string?

Reply:
(like, seems like this should replace the markDownPostEditor editor in a more generic way)

(also I'm suddenly having a moment of sadness for Tilia, who has spent the past two wednesdays working on a custom component to change the "Use Markdown Editor" boolean into a Select component. Alas. Shoulder-Genghis-Khan says to sacrifice her work in the name of progress)

Reply:
This is not the branch to do fancy refactor's in, given it's likelihood of potentially being ripped out.